api/allIncidents/incidentsModel.js

```diff
@@ -33,54 +35,98 @@ function getIncidentById(id) {
  * Returns incidents in the database sorted by newest incident first limited by the number defined in limit parameter
  */
 async function getTimelineIncidents(limit) {
-  return await db('incidents')
-    .whereNot({ date: null })
-    .orderBy('date', 'desc')
+  const incidents = await db('incidents')
+    .where({ status: 'approved' })
+    .whereNot({ date_created: null })
+    .orderBy('date_created', 'desc')
```

Comment:
Same `desc`-`description` concern as above.

Reply:
This desc means descending